.travis.yml

```diff
@@ -1,11 +1,19 @@
 language: node_js
 node_js:
-  - "8"
-  - "9"
+- '8'
+- '9'
+- '10'
 script:
-  - yarn test
-
+- yarn test
 notifications:
   email:
     on_success: never
     on_failure: always
+deploy:
```

Comment:
This causes Travis to automatically publish to NPM on a tagged commit (similar to how we use dev-tools). This allows us to update the Travis environment to publish as `lifeomicdev` rather than our individual accounts.